mate-about/mate-about.desktop.in

```diff
@@ -1,6 +1,6 @@
 [Desktop Entry]
-_Name=About MATE
-_Comment=Learn more about MATE
+Name=About MATE
```

Comment:
Why removing translations here?

Reply:
Ok, this is mentioned in docs. But how will it be translated?

Reply:
With gettext instead of intltool you do not need the _ prefix to translate.

Reply:
good